### DISCUSSION TITLE
Ubuntu minimal image

### DISCUSSION THREAD

**Discussion by andreaswittig** [community/community#160644] [Actions] [answered]
Hi,

I've noticed the `actions/runner-images` repository contains a template to build an Ubuntu minimal image. However, I could not find a way to use this image with GitHub-hosted runners. Is the Ubuntu minimal image for self-hosted runners only? Or am I missing something?

Thanks,
Andreas

**Comment by erik-bershel** [comment, marked as answer]
Hey @andreaswittig!

> Is the Ubuntu minimal image for self-hosted runners only?

Yeah, you are right, it's basic template to build self-hosted images only. There is **no** `ubuntu-minimal` image in GitHub.

**Reply by oliviergaraud** [reply]
Is there any plan to have so ? Or do we have an action to free disk space and remove software we don't need ?

**Reply by erik-bershel** [reply]
> Is there any plan to have so ?

I don't know about such plans. We'll inform community if something changed.

> Or do we have an action to free disk space and remove software we don't need ?

I don't know about such actions. But the marketplace is full of different useful tools, I can only recommend to check here.

**Reply by alxpsr** [reply]
sad(
